subprojects/gradle/module-dependencies-graph/src/main/kotlin/com/avito/module/internal/dependencies/AndroidAppsGraphBuilder.kt

```diff
@@ -6,17 +6,10 @@ internal class AndroidAppsGraphBuilder(
     private val graphBuilder: DependenciesGraphBuilder
 ) {
 
-    fun buildDependenciesGraph(type: ConfigurationType): Set<ModuleProjectConfigurationDependenciesNode> {
+    fun buildDependenciesGraph(type: ConfigurationType): Set<ProjectConfigurationNode> {
         return graphBuilder.buildDependenciesGraph(type)
             .filter { node ->
                 node.project.plugins.hasPlugin("com.android.application")
             }.toSet()
     }
-
-    fun buildDependenciesGraphFlatten(type: ConfigurationType): List<ProjectWithDeps> {
```

Comment:
Remove from Android graph. Here is nothing about Android, only collecting data from nodes.